.github/workflows/backend.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           key: ${{ runner.os }}-maven
       - name: Check Style
         run: |
-          ./mvnw -B checkstyle:check
+          ./mvnw -T 16 -B checkstyle:check
```

Comment:
hi @asdf2014, 16 threads is too big, let's use `-T 2C`.

Reply:
Sure, let me address

Reply:
@leo65535 Actually, the `-T 16` option is suitable, when the number of our connectors becomes larger, it will be very necessary, and 16 threads is used by other projects as well, for example, Apache Pinot. Anyway, I reduced this number of threads for now, and will raise up another PR to increase it when it is not enough later.